drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -551,6 +615,10 @@ static int bq274xx_gauge_init(const struct device *dev)
 	}
 #endif
 
+	while (k_uptime_get_32() < POWER_UP_DELAY_MS) {
+		k_sleep(K_MSEC(1));
+	}
```

Comment:
```suggestion
	int32_t delay_remainder = POWER_UP_DELAY_MS - k_uptime_get_32();
	if (delay_remainder > 0) {
		LOG_DBG("power up delay remainder: %d", delay_remainder);
		k_msleep(delay_remainder);
	}
```

Reply:
cool I like it, tweaked the variable name but used your code otherwise